templates/server-route.yaml

```diff
@@ -14,20 +15,25 @@ metadata:
     app.kubernetes.io/name: {{ include "vault.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    {{- with .Values.server.route.labels }}
+    {{- with $route.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
   {{- template "vault.route.annotations" . }}
 spec:
-  host: {{ .Values.server.route.host }}
+{{- if $route.host }}
+  host: {{ $route.host }}
+{{- end }}
   to:
     kind: Service
     name: {{ $serviceName }}
     weight: 100
   port:
     targetPort: 8200
+  {{- if $route.tls.enabled }}
   tls:
-    termination: passthrough
+    insecureEdgeTerminationPolicy: {{ $route.tls.insecureEdgeTerminationPolicy }}
+    termination: {{ $route.tls.termination }}

```

Comment:
we will need to provide tls termination policy in values.yaml, seems like its  missing

Reply:
What should I provide. The line in red ?